src/data/proto/envoy/annotations/deprecation.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable */
+
+export const protobufPackage = "envoy.annotations";
```

Comment:
There are extensions that makes github diffs clever and not show entire diffs for generated files. We should be using one of these extensions to make reviews easier. I think it's https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

Reply:
Ah yes definitely! If the generated types come in here, we could add that. Do you have any thoughts on who would own the auto generation of the ts files, i.e. would you want the frontend to own the scripts that do this, or would the backend own those?

Reply:
I think GUI would own these.